dspace-api/src/main/java/org/dspace/authenticate/ShibAuthentication.java

```diff
@@ -1258,6 +1246,28 @@ protected String findSingleAttribute(HttpServletRequest request, String name) {
         return valueList;
     }
 
+    private String getShibURL(HttpServletRequest request) {
+        String shibURL = configurationService.getProperty("authentication-shibboleth.lazysession.loginurl");
+        boolean forceHTTPS =
+                configurationService.getBooleanProperty("authentication-shibboleth.lazysession.secure",true);
+
+        // Shibboleth authentication initiator
+        if (shibURL == null || shibURL.length() == 0) {
+            shibURL = "/Shibboleth.sso/Login";
+        }
+        shibURL = shibURL.trim();
```

Comment:
A small comment.  You don't need to do all this string parsing of `shibURL`, as the ConfigurationService is much "smarter" than the old ConfigurationManager.  The string will already be trimmed.  Additionally, the `if` just before this is unneeded, as you can pass in a default value if it's unspecified.  So, both the `if` and the `trim()` can be removed in favor of just calling

`String shibURL = configurationService.getProperty("authentication-shibboleth.lazysession.loginurl", "/Shibboleth.sso/Login");`

Reply:
done